pkg/gatewayserver/io/io.go

```diff
@@ -33,7 +33,9 @@ import (
 
 const (
 	bufferSize = 1 << 4
-	maxRTTs    = 1 << 5
+
```

Comment:
I think we should expose this a config. We can file an issue for this to target `develop`.

Reply:
Yeah, let's do that as-needed